client/src/leap/soledad/client/encdecpool.py

```diff
@@ -212,29 +174,7 @@ def enqueue_doc_for_encryption(self, doc):
         :param doc: The document to be encrypted.
```

Comment:
Renamed! I think I got so used to "enqueue" stuff on this module that I didn't realize queue is gone. Also merged with the other method.